R/expr__expr.R

```diff
@@ -3066,13 +3066,14 @@ Expr_rolling_std = function(
     min_periods = NULL,
     center = FALSE, # :bool,
     by = NULL, # : Nullable<String>,
-    closed = "left" # ;: Nullable<String>,
+    closed = c("left", "right", "both", "none")
     ) {
   wargs = prepare_rolling_window_args(window_size, min_periods)
-  unwrap(.pr$Expr$rolling_std(
+  .pr$Expr$rolling_std(
     self, wargs$window_size, weights,
-    wargs$min_periods, center, by, closed
-  ))
+    wargs$min_periods, center, by, closed[1L]
```

Comment:
missing before merge, roll out this solution to all. Consider not using [1L] subsetting but simply pass as is to rust side. Likely extendr as_str() don't care if it is a string or a char vec, and will only access first element.

Reply:
will get back to [1L] in new PR